Sources/ProjectDescription/Package.swift

```diff
@@ -0,0 +1,245 @@
+public enum Package: Equatable, Codable {
```

Comment:
Since this is now on top level and user will see documentation for this, I'd maybe add a little comment for this

Reply:
How about:
```
/// A dependency of a Swift package.
///
/// A package dependency can be either:
///     - remote: A Git URL to the source of the package,
///     and a requirement for the version of the package.
///     - local: A relative path to a swift package.
```

Reply:
Looks good 👍